tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -155,6 +155,14 @@ export class WebGPUBackend extends KernelBackend {
         type: 'timestamp',
         count: 2,
       });
+    } else {
+      console.warn(
+          `This device doesn't support timestamp-query extension. ` +
+          `Start Chrome browser with flag ` +
+          `--disable-dawn-features=disallow_unsafe_apis then try again. ` +
+          `Or zero will shown for the kernel time when profiling mode is` +
+          `enabled. Using performance.now is not workable for webgpu since` +
+          `it doesn't support synchronously to read data from GPU.`);
```

Comment:
I don't quite understand why we always need this warning. This is a development only extension.

Reply:
Just noticed a bug https://github.com/tensorflow/tfjs/issues/6691 related with it. Agree with Yang's opinion. We should only print this warning when the users really use it.